packages/common/src/utils/formatUtil.ts

```diff
@@ -180,6 +181,12 @@ export const parseWeiNumber = (number: string) => {
   }
 }
 
+// TODO convert to new fn
+export const formatUSDC = (number: StringUSDC) => {
```

Comment:
This is merged now if you'd like to pull in latest from the feature branch and use it. The function is in `utils/wallet.ts`